liquidity/liquidity.go

```diff
@@ -420,6 +510,71 @@ func (m *Manager) makeLoopOutRequest(suggestion *LoopOutRecommendation,
 	}
 }
 
+// calculateExpenditure calculates the total amount that has been spent by
+// automatically dispatched swaps. If there is an automatically dispatched swap
+// currently in progress, this function will fail with errAutoSwapInFlight
+// because we cannot accurately calculate off-chain fees for an in-flight
+// swap.
+func (m *Manager) calculateExpenditure(ctx context.Context,
```

Comment:
nit: should be called `calculateOutExpenditure` or will it also return loop in fees in the future? If the latter is true, maybe we could already include the return parameter but always return it as 0.